package.json

```diff
@@ -4,32 +4,44 @@
   "description": "",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/cozy/cozy-konnector-maif.git"
+    "url": "git+https://github.com/konnectors/cozy-konnector-maif.git"
   },
   "keywords": [],
   "author": "",
   "license": "AGPL-3.0",
-  "dependencies": {
-    "cozy-konnector-libs": "3.2.5"
+  "main": "src/index.js",
+  "renovate": {
+    "extends": ["config:base"],
+    "patch": {
+      "automerge": true
+    }
   },
   "scripts": {
     "test": "jest",
-    "start": "node index.js",
-    "dev": "cozy-konnector-dev",
-    "standalone": "cozy-konnector-standalone",
-    "standalone:record": "REPLAY=record cozy-konnector-standalone",
-    "standalone:replay": "REPLAY=replay cozy-konnector-standalone",
+    "start": "node src/index.js",
+    "dev": "cozy-konnector-dev src/index.js",
+    "standalone": "cozy-konnector-standalone src/index.js",
+    "standalone:record": "REPLAY=record cozy-konnector-standalone src/index.js",
+    "standalone:replay": "REPLAY=replay cozy-konnector-standalone src/index.js",
     "build": "webpack",
+    "format": "prettier --write '**/*.{js,json}'",
+    "precommit": "pretty-quick --staged",
     "lint": "standard --fix index.js",
-    "deploy": "git-directory-deploy --directory build/ --branch build --repo=https://github.com/cozy/cozy-konnector-maif.git",
-    "deploytest": "git-directory-deploy --directory build/ --branch buildtest --repo=https://github.com/cozy/cozy-konnector-maif.git",
-    "deploy:travis": "git-directory-deploy --username Cozy --email contact@cozycloud.cc --directory build/ --repo=https://$GITHUB_TOKEN@github.com/cozy/cozy-konnector-maif.git"
+    "deploy":
```

Comment:
I guess those lines should go into cozy-scripts as we reuse them across projects.